notifications/smart_reminder_serializers.py

```
@@ -0,0 +1,390 @@
+        
+        for time_str in value:
+            try:
+                hour, minute = time_str.split(':')
+                hour = int(hour)
+                minute = int(minute)
+                if not (0 <= hour <= 23 and 0 <= minute <= 59):
+                    raise ValueError
+            except (ValueError, AttributeError):
+                raise serializers.ValidationError(
+                    f"فرمت زمان نامعتبر: {time_str}. فرمت صحیح: HH:MM"
+                )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)